locations/spiders/caesars.py

```diff
@@ -5,7 +5,11 @@
 
 class CaesarsSpider(scrapy.Spider):
     name = "caesars"
-    item_attributes = {"brand": "Caesars Entertainment", "brand_wikidata": "Q18636524"}
+    item_attributes = {
+        "brand": "Caesars Entertainment",
+        "brand_wikidata": "Q18636524",
+        "extras": {"leisure": "resort", "amenity": "casino"},
```

Comment:
I really don't like multiple categories, it makes everything harder. I vote for:
```suggestion
        "extras": {"amenity": "casino"}
```

As you said, it's more gambling than hotel. And leisure=resort looks more... fun, the wiki mentions water slides and ski slopes, not gambling.